automation_tools/__init__.py

```diff
@@ -476,7 +476,7 @@ def setup_default_subnet(sat_version):
         '--gateway {gateway} --dns-primary {gateway} '
         '--ipam DHCP --from {from} --to {to} '
         '--dhcp-id 1 --dns-id 1 --tftp-id 1 ' +
-        ('--discovery-id 1' if sat_version > '6.2' else '')
```

Comment:
This is updated as sat_version can also be a string like `upstream-nightly` or `downstream-nightly`